### PR TITLE
Never upload with setup.py

### PR DESCRIPTION
**Pull request by Carreau**
It **does** upload without checking the certificate of the remote
server, does not use ssl, and send the credential in plaintext on the
wire.

---- 

See https://packaging.python.org/distributing/, for longer description, you will need to install `twine` of course. 

I would recommend checking https://packaging.python.org/distributing/?highlight=twine#create-an-account that ask to update your `.pypirc` to : 

```
[distutils]
index-servers=pypi

[pypi]
repository = https://upload.pypi.io/legacy/
username = <username>
password = <password>
```

In order to use the new endpoint. The old endpoint have a know ~5-10% upload failure.

### CODE REVIEW

**Comment by blink1073** [comment]
I think this entire part should be removed from the readme, in favor of a developer readme that has the broader explanation that you gave above.

**Comment by jasongrout** [comment]
(setting to bug since it is a security issue to upload via setup.py)

File: README.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ We publish an npm package, a Python source package, and a Python universal binar
 npm version patch
 git push origin master --tags
 npm publish
```
Comment (jasongrout) [inline README.md]: `rm -rf dist`, so that twine upload only uploads what you just built....

**Comment by blink1073** [comment]
I'm going to merge and iterate on this, thanks @Carreau!

**Comment by Carreau** [comment]
Sorry I have seen the notifications, but have difficulty catching up with my backlog. 

Thanks **a lot** for taking care of that. 

I've also seen that a lot of issues that I open have been fixed and is really looking forward to have time to update, and have occasion to use the notebook ! Hopefully I'll use it for PyBay !

**Pull request merged**